code/modules/clothing/masks/gasmask.dm

```diff
@@ -386,3 +386,25 @@
 	flags_inv = HIDEFACIALHAIR
 	w_class = WEIGHT_CLASS_NORMAL
 	inhand_icon_state = "driscoll_mask"
+
+/obj/item/clothing/mask/gas/prop
+	name = "prop gas mask"
+	desc = "A prop gas mask designed for appearance. Unlike a normal gas mask this does not filter gasses or protect against pepper spray."
+	icon_state = "gas_alt"
+	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
+	w_class = WEIGHT_CLASS_SMALL
+	inhand_icon_state = "gas_alt"
+	permeability_coefficient = 0.5
+	flags_cover = MASKCOVERSEYES
```

Comment:
And this

Reply:
Removing the flags_cover will change it back to being pepper proof

Reply:
Set it to NONE